src/WingetCreateCLI/FieldValidation.cs

```diff
@@ -31,40 +32,43 @@ public static class FieldValidation
                 }
 
                 var validationContext = new ValidationContext(instance) { MemberName = memberName };
+                Type type = instance.GetType().GetProperty(memberName).PropertyType;
+                List<ValidationResult> validationResults = new List<ValidationResult>();
 
-                if (defaultValue != null && defaultValue is not string && typeof(IEnumerable<string>).IsAssignableFrom(defaultValue.GetType()))
+                if (typeof(IEnumerable<string>).IsAssignableFrom(type) || (defaultValue != null && typeof(IEnumerable<string>).IsAssignableFrom(defaultValue.GetType())))
                 {
-                    // If prompt is an IEnumerable<string>, validate each item in the enumerable against the property
-
                     // If the user didn't provide a value, check if null is allowed for field
                     var items = property == null
-                        ? new string[] { null }
-                        : (property as string).Split(',', StringSplitOptions.RemoveEmptyEntries | StringSplitOptions.TrimEntries);
+                        ? new List<string>()
```

Comment:
I may be missing why you changed this, but I needed the null in the array so that that value was checked against the field requirements on line 50, for InstallerUrls

Reply:
I accidentally deleted the null, I also needed that. 

I changed the array to a list as I was having conflicts with the types when dealing with Tags which is a List<string>